recipes/spaghetti-bolognese.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <h1>Spaghetti Bolognese</h1>
-    <img src="images/spaghettibolognese.jpeg">
```

Comment:
make sure you dont have any unneeded files in images folder

Reply:
addressed